raster/src/main/scala/geotrellis/raster/RasterExtent.scala

```diff
@@ -241,10 +241,10 @@ object RasterExtent {
     RasterExtent(extent, cw, ch, cols, rows)
   }
 
-  def apply(extent: Extent, cellwidth: Double, cellheight: Double): RasterExtent = {
-    val cols = (extent.width / cellwidth).toInt
-    val rows = (extent.height / cellheight).toInt
-    RasterExtent(extent, cellwidth, cellheight, cols, rows)
+  def apply(extent: Extent, cellSize: CellSize): RasterExtent = {
```

Comment:
Any reason not to have both as overloads? This might break some things, and unless there's a good reason to prevent the overload that takes extent double double I'd prefer to keep it.